x-vector/src/X/Data/Vector/Generic.hs

```diff
@@ -14,12 +22,19 @@ module X.Data.Vector.Generic (
   , mapMaybeM
   , imapMaybeM
 
+  -- * Modifying vectors
+
+  -- ** Transposition
+  , transpose
+
```

Comment:
cool, yeah if you ever had some other kind of vector which could be on the outside, you can always import from Generic